config/install.yaml

```diff
@@ -1868,26 +1949,6 @@ spec:
         volumeMounts:
         - mountPath: /etc/numaplane
           name: config-volume
-      initContainers:
-      - command:
-        - /bin/bash
-        - -c
-        - "#!/bin/bash\n\n# Get all namespaces\nnamespaces=$(kubectl get namespaces
-          -o jsonpath='{.items[*].metadata.name}')\n\n# Loop through each namespace\nfor
-          namespace in $namespaces; do\n  echo \"Processing namespace: $namespace\"\n
-          \ \n  # Get all isbsvc in the current namespace\n  interstepbuffersvcs=$(kubectl
-          get isbsvc -n $namespace -o json | jq -r '.items[] | select(.metadata.ownerReferences[]?.kind
-          == \"ISBServiceRollout\") | .metadata.name')\n  \n  # Loop through each
-          isbsvc\n  for isbsvc in $interstepbuffersvcs; do\n    echo \"Patching isbsvc:
-          $isbsvc in namespace: $namespace\"\n    \n    # Patch the isbsvc to add
-          the labels\n    kubectl patch isbsvc $isbsvc -n $namespace --type='merge'
-          -p \"{\\\"metadata\\\": {\\\"labels\\\": {\\\"numaplane.numaproj.io/upgrade-state\\\":
-          \\\"promoted\\\"}}}\"\n    kubectl patch isbsvc $isbsvc -n $namespace --type='merge'
-          -p \"{\\\"metadata\\\": {\\\"labels\\\": {\\\"numaplane.numaproj.io/parent-rollout-name\\\":
-          \\\"$isbsvc\\\"}}}\"\n  done\ndone\n\necho \"All interstepbufferservices
-          have been patched.\"\n"
-        image: bitnami/kubectl
-        name: label-isbsvc
```

Comment:
This initContainer was added to numaplane 0.12.0, which is being rolled out now to clusters. It helped the transition to the use of new labels "parent-rollout-name" and "upgrade-state" for isbsvc in the clusters without backward compatibility issues. However, before we actually have blue/green operational, we need to remove this Initcontainer since it will create the wrong label for "parent-rollout-name".

Reply:
Therefore, we must release numaplane 0.12.0 (or otherwise some patch 0.12.x) to all clusters which adds the labels prior to rolling this PR (which removes the initcontainer) out to clusters.